.circleci/config.yml

```diff
@@ -282,29 +266,42 @@ jobs:
       - run:
           name: Build and run tests
           no_output_timeout: 35m
-          command: cd utilities && RUST_MIN_STACK=8388608 cargo test
+          command: cd utilities && RUST_MIN_STACK=67108864 cargo test
       - clear_environment:
           cache_key: snarkvm-utilities-cache
 
+  fmt:
+    docker:
+      - image: howardwu/snarkos-ci:2021-03-25
```

Comment:
```suggestion
      - image: cimg/rust:1.53.0
```

Reply:
Oh nope! that image is for the `nightly` because our `fmt` is done over it.